src/select/select.ts

```diff
@@ -259,6 +259,7 @@ export class SelectComponent implements OnInit, ControlValueAccessor {
   @Input() public textField:string = 'text';
   @Input() public childrenField:string = 'children';
   @Input() public multiple:boolean = false;
+  @Input() public formControl = new FormControl(); 
```

Comment:
I'm pretty sure I tried that and it didn't work. I'll try to test it again as well.